main/source/mod/AvHClientVariables.h

```diff
@@ -48,6 +48,7 @@ extern cvar_t*	cl_musicdir;
 #define	kvCMHotKeys			"cl_cmhotkeys"
 #define	kvForceDefaultFOV	"cl_forcedefaultfov"
 #define	kvWidescreen		"cl_widescreen"
+#define	kvAmbientSound		"cl_ambientsound"
```

Comment:
we should set standards around things like spacing (space vs tab) and e.g. always curly braced ifs.

Reply:
mixed spacing like this will look strange for people with different tab widths etc